libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -276,15 +275,28 @@ export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvide
      * });
      * ```
      */
-    public async continueConversation(reference: Partial<ConversationReference>, logic: (context: TurnContext) => Promise<void>): Promise<void> {
-        const request: Partial<Activity> = TurnContext.applyConversationReference(
+    public async continueConversation(claimsIdentity: ClaimsIdentity, reference: Partial<ConversationReference>, logic: (context: TurnContext) => Promise<void>): Promise<void>;
+    public async continueConversation(reference: Partial<ConversationReference>, logic: (context: TurnContext) => Promise<void>): Promise<void>;
+    public async continueConversation(...args): Promise<void> {
+        let claimsIdentity, reference, logic;
+        if (args.length === 3) {
```

Comment:
It would be great to generalize this logic for the other overloads, but that could be done in a different milestone.

If we take a laissez faire (more JS-like?) approach for this logic, should we instead do `args.length <== 3`? Otherwise if we continue with the current code I feel we should probably throw an Error.

Reply:
Not sure what you are trying to achieve.  if args.length is < 3 we want the else to fire...

Reply:
D'oh, `args.length >== 3`

Reply:
will do.